premium/offline_test.go

```diff
@@ -153,12 +162,76 @@ func licenseTest(inputPath string, meta plugin.Meta, timeIs time.Time, expectErr
 		timeFunc = func() time.Time {
 			return timeIs
 		}
-
-		err := ValidateLicense(zerolog.Nop(), meta, inputPath)
+		licenseClient, err := NewLicenseClient(context.TODO(), zerolog.Nop(), WithMeta(meta), WithLicenseFileOrDirectory(inputPath))
+		require.NoError(t, err)
+		err = licenseClient.ValidateLicense(context.TODO())
 		if expectError == nil {
 			require.NoError(t, err)
 		} else {
 			require.ErrorIs(t, err, expectError)
 		}
 	}
 }
+
+func TestValidateMarketplaceLicense(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	m := mocks.NewMockAWSLicenseManagerInterface(ctrl)
+	out := licensemanager.CheckoutLicenseOutput{}
+	in := licenseInput{
+		CheckoutLicenseInput: licensemanager.CheckoutLicenseInput{
+			CheckoutType: types.CheckoutTypeProvisional,
+			ProductSKU:   aws.String("55ukc0d5qv3gebks148tjr62j"),
```

Comment:
could've used the const here too.

Reply:
[f2d4386](https://github.com/cloudquery/plugin-sdk/pull/1889/commits/f2d4386e997ebd8dc4bfb67cbb2d37d18d13287e)